model/link_metadata.go

```diff
@@ -17,10 +19,82 @@ const (
 	LINK_METADATA_TYPE_IMAGE     LinkMetadataType = "image"
 	LINK_METADATA_TYPE_NONE      LinkMetadataType = "none"
 	LINK_METADATA_TYPE_OPENGRAPH LinkMetadataType = "opengraph"
+	MAX_IMAGES                   int              = 5
 )
 
 type LinkMetadataType string
 
+// checks if original string has more than 300 characters long, if so
```

Comment:
This is a bit of nitpicking, but:
1. Can you move these function below the definition of `LinkMetadata`? We usually define types at the top of the file in this package
2. Golint recommends having the name of the function at the start of these docs. We could also make this more concise. Like these comments could be

        truncateText truncates the text of OpenGraph metadata to be 300 characters long and adds ellipses if necessary.